CONTRIBUTING.rst

```diff
@@ -624,6 +646,12 @@ Here's a more detailed flowchart of how to submit changes.
    A pull request is essentially just asking people to review and accept the
    modifications you have made to your personal version of the code.
 
+#. Once you pull request is merged, stay up to date with the main yt repository
+   by pulling from the ``upstream`` remote::
+
+     git checkout master
+     git pull upstream master
```

Comment:
`git branch -d my-first-pull-request`
to delete your feature branch locally (optional)